addons/website/static/src/systray_items/edit.js

```diff
@@ -0,0 +1,117 @@
+/** @odoo-module **/
```

Comment:
This is a merge of edit_website.js and edit_in_backend.js + I guess a few changes you made, this is making the review really more difficult than it should IMO.
Could you please make a first commit where you merge those 2 files, it can be a temp commit during the review or even a [MOV] commit that we will keep (and it will "save" the git history of one of the 2 files)

Probably same for the xml file but less important as quite small
Same for the website_blog file